README.md

```diff
@@ -36,3 +36,22 @@ $ rails server
 ```
 
 Visit `http://localhost:3000/`.
+
+
+### MySQL Setup
+
+Webscale is configured to use MySQL as it's database. In macOS, MySQL can be installed using brew:
+
+```bash
+$ brew install mysql
+```
+
+By default the mysql user is root with no password.
+
+Once installed, verify your environments connection to the database by creating the databases:
+
+```
+$ rake db:create
```

Comment:
I had to do `mysql.server start` before this because mysql wasn't running for me.